### ISSUE TITLE
create and use a fork of react-script with newer dependencies

### ISSUE THREAD

**Issue by magwas** [kode-konveyor/taskmarket-frontend#23]
We use much more up-to-date versions of several packages than react-script.
Fork the https://github.com/facebook/create-react-app repo, fix the react-scripts package using the knowledge found in the current fixup script:
[fixups.zip](https://github.com/kode-konveyor/taskmarket-frontend/files/4971856/fixups.zip)
Use the fork as a dependency to the project.

**Comment by Mayuresh94** [comment]
is this task to update "dependencies" in package.json of this project referring fixup?

**Comment by magwas** [comment]
No, this is about forking the react project, and fixing the dependencies inside it.
